spec/lib/orbf/rules_engine/services/ethiopian_calendar_spec.rb

```diff
@@ -0,0 +1,47 @@
+require "date"
+
+RSpec.describe Orbf::RulesEngine::EthiopianCalendar do
```

Comment:
Thank you for this, its' really hurting my head.

Do you know if this will handle leap years as well?

Reply:
In the original code the leap year conditions where for the julian day number conversions for gregorian dates that are now using the `Date.jd` that I assumed "leap safe".